book/src/rust-client/mint_consume_send_tutorial.md

```diff
@@ -0,0 +1,450 @@
+# Mint, Consume, and Send Assets
+
+*Using the Miden Client in Rust to Mint, Consume, and Send Assets*
+
+## Overview
+
+In the previous section we initialized our repository and covered how to create an account and deploy a faucet. In this section we will mint tokens from the faucet for *Alice*, consume these created notes, and then show how to send assets to other accounts.
+
+## What We'll Cover
+* Minting tokens from a faucet
+* Consuming notes
+* Sending tokens to other users
+
+## Step 1: Minting Tokens from the Faucet
+To mint notes with tokens from the faucet we created, Alice needs to call the faucet with a mint transaction request.
+
+```rust
+let amount: i32 = 100;
+let fungible_asset = FungibleAsset::new(faucet_account.id(), amount)
+    .unwrap();
+
+let transaction_request = TransactionRequest::mint_fungible_asset(
+    fungible_asset.clone(), // fungible asset id
+    alice_account.id(),     // target account id
+    NoteType::Public,       // minted note type
+    client.rng(),           // rng 
+)
+.unwrap();
+
+let tx_execution_result = client
+    .new_transaction(faucet_account.id(), transaction_request)
+    .await?;
+
+
+client.submit_transaction(tx_execution_result).await?;
+```
+
+## Step 2: Identifying Consumable Notes
+Once Alice has minted a note from the faucet, she will eventually want to spend the tokens that she received in the note created by the mint transaction. To identify notes that are ready to consume, the miden-client has a useful function `get_consumable_notes`. It is also important to sync the state of the client before calling the `get_consumable_notes` function. 
+
+*Tip: If you know how many notes to expect after a transaction, use an await or loop condition to check how many notes of the type you expect are available for consumption instead of using a set timeout before calling `get_consumable_notes`. This ensures your application isn't idle for longer than necessary.*
+
+#### Identifying which notes are available:
+
+```rust
+let consumable_notes = client.get_consumable_notes(Some(alice_account.id())).await?;
+```
+
+## Step 3: Consuming multiple notes in a single transaction:
+Now that we've identified the notes ready to consume, we can consume multiple notes in a single transaction. After consuming the notes, Alice's wallet balance will be updated.
+
+```Rust
+let transaction_request = TransactionRequest::consume_notes(list_of_note_ids);
```

Comment:
In our example there would be just one note, right? At least we minted only once in the code above. Or am I missing something?